packages/plugin-core/src/WorkspaceWatcher.ts

```diff
@@ -82,9 +81,9 @@ export class WorkspaceWatcher {
       return { changes: [] };
     }
     const match = NoteUtils.RE_FM_UPDATED.exec(content);
-    const noteHash = genHash(content);
     let changes: TextEdit[] = [];
-    if (match && note.contentHash && note.contentHash !== noteHash) {
+
+    if (match && WorkspaceUtils.noteContentChanged({ content, note })) {
```

Comment:
I tried this branch in #843 and after a save `note.contentHash` is still `undefined`. When I run `Dendron: Reload Index` it works only the first time

Reply:
you're right, there was an issue on saving hash with new content. just fixed with new push

Reply:
awesome, this works now!